docker/php-fpm/Dockerfile

```diff
@@ -34,7 +34,7 @@ WORKDIR /app/src
 
 COPY --chown=php:php --from=builder /app/src/vendor /app/src/vendor
 COPY --chown=php:php docker/php-fpm /
-COPY src/ /app/src/
+COPY --chown=php:php src/ /app/src/
```

Comment:
This ensures that all the project files are owned by the php user in the container